Makefile

```diff
@@ -6,15 +6,16 @@ GIT_COMMIT = $(shell git rev-parse HEAD)
 DATE = $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
 
 TARGET ?= local
+KIND_NAME ?= kind
 
-KIND_KUBE_CONFIG_FOLDER = $${HOME}/.kube/kind
+KIND_KUBE_CONFIG_FOLDER = $${HOME}/.kube/$(KIND_NAME)
 OUT_DIR=$(shell pwd)/build/.out
 
 .DEFAULT_GOAL := help
 
 .PHONY: help
 help: Makefile ## Display this help
-	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "; printf "Usage:\n\n    make \033[36m<target>\033[0m\n\nTargets:\n\n"}; {printf "    \033[36m%-30s\033[0m %s\n", $$1, $$2}'
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "; printf "Usage:\n\n    make \033[36m<target>\033[0m KIND_NAME=\033[36m<cluster-name>\033[0m\n\nTargets:\n\n"}; {printf "    \033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Comment:
Rather than including this in the usage (since it makes it seem like KIND_NAME is included for any make target), can you update `docs/developer/quickstart.md#deploy-on-kind` to mention this option?